internal/api/api.go

```diff
@@ -100,23 +105,52 @@ func (api *idler) Idle(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 }
 
 func (api *idler) UnIdle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	openShiftAPI, openShiftBearerToken, err := api.getURLAndToken(r)
+
+	openshiftURL, openshiftToken, err := api.getURLAndToken(r)
 	if err != nil {
-		log.Error(err)
-		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte(fmt.Sprintf("{\"error\": \"%s\"}", err)))
+		respondWithError(w, http.StatusBadRequest, err)
```

Comment:
Whynot log the error as well? If the consumer does not log the exception, nor show to the user, we will not be able to know what's happening, isn't it?

Reply:
nvm my previous comment